extension/src/time_series/pipeline/arithmetic.rs

```diff
@@ -0,0 +1,638 @@
+            ]");
+
+            let val = client.select(
+                &format!("SELECT (series |> div(5.0))::TEXT FROM ({}) s", create_series),
+                None,
+                None
+            )
+                .first()
+                .get_one::<String>();
+            assert_eq!(val.unwrap(), "[\
+                {\"ts\":\"2020-01-04 00:00:00+00\",\"val\":5.0},\
+                {\"ts\":\"2020-01-01 00:00:00+00\",\"val\":2.0},\
+                {\"ts\":\"2020-01-03 00:00:00+00\",\"val\":4.0},\
+                {\"ts\":\"2020-01-02 00:00:00+00\",\"val\":3.0},\
+                {\"ts\":\"2020-01-05 00:00:00+00\",\"val\":6.0}\
+            ]");
+
+            let val = client.select(
+                &format!("SELECT (series |> mod(5.0))::TEXT FROM ({}) s", create_series),
+                None,
+                None
+            )
+                .first()
+                .get_one::<String>();
+            assert_eq!(val.unwrap(), "[\
+                {\"ts\":\"2020-01-04 00:00:00+00\",\"val\":0.0},\
+                {\"ts\":\"2020-01-01 00:00:00+00\",\"val\":0.0},\
+                {\"ts\":\"2020-01-03 00:00:00+00\",\"val\":0.0},\
+                {\"ts\":\"2020-01-02 00:00:00+00\",\"val\":0.0},\
+                {\"ts\":\"2020-01-05 00:00:00+00\",\"val\":0.0}\
+            ]");
+
+            let val = client.select(
+                &format!("SELECT (series |> power(2.0))::TEXT FROM ({}) s", create_series),
+                None,
+                None
+            )
+                .first()
+                .get_one::<String>();
+            assert_eq!(val.unwrap(), "[\
+                {\"ts\":\"2020-01-04 00:00:00+00\",\"val\":625.0},\
+                {\"ts\":\"2020-01-01 00:00:00+00\",\"val\":100.0},\
+                {\"ts\":\"2020-01-03 00:00:00+00\",\"val\":400.0},\
+                {\"ts\":\"2020-01-02 00:00:00+00\",\"val\":225.0},\
+                {\"ts\":\"2020-01-05 00:00:00+00\",\"val\":900.0}\
+            ]");
+
+
+            let val = client.select(
+                &format!("SELECT (series |> logN(10.0))::TEXT FROM ({}) s", create_series),
+                None,
+                None
+            )
+                .first()
+                .get_one::<String>();
+            assert_eq!(val.unwrap(), "[\
+                {\"ts\":\"2020-01-04 00:00:00+00\",\"val\":1.3979400086720375},\
+                {\"ts\":\"2020-01-01 00:00:00+00\",\"val\":1.0},\
+                {\"ts\":\"2020-01-03 00:00:00+00\",\"val\":1.301029995663981},\
+                {\"ts\":\"2020-01-02 00:00:00+00\",\"val\":1.1760912590556811},\
+                {\"ts\":\"2020-01-05 00:00:00+00\",\"val\":1.4771212547196624}\
+            ]");
+        });
+    }
+
+    #[pg_test]
+    fn test_simple_arith_unaryops() {
+        Spi::execute(|client| {
+            client.select("SET timezone TO 'UTC'", None, None);
+            // using the search path trick for this test b/c the operator is
+            // difficult to spot otherwise.
```

Comment:
I thought we'd decided that the operator was going to go in the non-schema qualified part?